src/components/Calculator.js

```diff
@@ -0,0 +1,40 @@
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <div className="Calc">
+      <div className="row1 Display">0</div>
+      <div className="row2">
+        <button type="button" className="Calc-button">AC</button>
+        <button type="button" className="Calc-button">+/-</button>
+        <button type="button" className="Calc-button">%</button>
+        <button type="button" className="Calc-button Calc-operator">/</button>
+      </div>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Calculator.js%20file.-,In%20Calculator.js%2C%20create%20a%20React%20component%20that%20matches%20this%20design%3A,-Implement%20the%20design), Please replace the divide symbol you added `/` with this symbol `÷` instead to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/218836370-5286a6b0-3103-4941-aea2-12914937bb0e.png)  | ![image](https://user-images.githubusercontent.com/68796232/218898152-c080f545-e05f-4e4a-860a-5eb45a24656f.png) |